tests/routes/search_test.py

```diff
@@ -90,6 +90,26 @@ def test_empty_search_path(self):
             }) \
             .post({}, headers=self.auth)
 
+    def test_sort_metadata_property_does_not_exist(self):
+        # Ensuring we don't blow up on a sort like this.
+        # This was a bug cataloged AGI-1346 and changelog release v6.1
```

Comment:
Could you create an issue for this instead of referencing an AGI number then reference your pull request in it?  This number means nothing to anyone who doesn't work for CWS.